examples/fedpca_examples/dim_reduction/README.md

```diff
@@ -0,0 +1,42 @@
+# PCA Dimensionality Reduction Example
+This example leverages federally computed principal components of the MNIST dataset to perform dimensionality reduction on the images, before proceeding with normal training.
+
+This example assumes that the principal components of MNIST have already been computed and saved (run the example in `exampes/fedpca_examples/perform_pca` to do this), and the user supplies a path to the saved principal components to perform dimensionality reduction.
+
+Each client performs Dirichlet subsampling on the whole dataset to produce heterogeneous local datasets.
+
+## Running the Example
+In order to run the example, first ensure you have the virtual env of your choice activated and run
+```
+pip install --upgrade pip
```

Comment:
Small point: Marcelo updated the installation instructions of the examples to reflect we are only using poetry for package management in the most recently merged PR. You can take a look what he did and just paste it in here

Reply:
Good point!